
project build/make/
diff --git a/tools/buildinfo.sh b/tools/buildinfo.sh
index 9a03fadc8..0886036e6 100755
--- a/tools/buildinfo.sh
+++ b/tools/buildinfo.sh
@@ -17,8 +17,8 @@ echo "ro.build.version.min_supported_target_sdk=$PLATFORM_MIN_SUPPORTED_TARGET_S
 echo "ro.build.date=`$DATE`"
 echo "ro.build.date.utc=`$DATE +%s`"
 echo "ro.build.type=$TARGET_BUILD_TYPE"
-echo "ro.build.user=$USER"
-echo "ro.build.host=`hostname`"
+echo "ro.build.user=$KBUILD_BUILD_USER"
+echo "ro.build.host=$KBUILD_BUILD_HOST"
 echo "ro.build.tags=$BUILD_VERSION_TAGS"
 echo "ro.build.flavor=$TARGET_BUILD_FLAVOR"
 if [ -n "$BOARD_BUILD_SYSTEM_ROOT_IMAGE" ] ; then
